packages/dendron-cli/src/commands/devCLICommand.ts

```diff
@@ -450,7 +450,7 @@ export class DevCLICommand extends CLICommand<CommandOpts, CommandOutput> {
     await this.syncAssets(opts);
 
     this.print("prep repo...");
-    await BuildUtils.prepPluginPkg();
```

Comment:
@jonathanyeung was there a reason why we didn't pass in the extension target directly at this phase?

Reply:
Your change should be fine - I think I had just done that to minimize chances of regressing the regular release build when I was first setting up the nightly stuff.